string/asmdefs.h

```diff
@@ -81,4 +81,18 @@ GNU_PROPERTY (FEATURE_1_AND, FEATURE_1_BTI|FEATURE_1_PAC)
 
 #define L(l) .L ## l
 
+  .macro ptr_param reg_num:req
+#ifdef __ILP32__
+  /* Sanitize padding bits of pointer arguments as per aapcs64 */
+  mov w\reg_num, w\reg_num
+#endif
+  .endm
```

Comment:
please use

#define DELOUSE(n)     mov     w##n, w##n

cpp macro instead of asm macros, because this is what we used in the ilp32 glibc and linux kernel patches. (i don't think it's a very good name but i prefer consistency here in case code will be moved between projects).

otherwise the patch looks ok, thanks.